### PR TITLE
Fix 'builtin-code' object has no attribute 'co_filename' under pypy

### PR DESCRIPTION
**Pull request by breezechen**

### CODE REVIEW

File: stackprinter/utils.py [hunk 1]
```diff
@@ -37,8 +37,16 @@ def inspect_callable(f):
...
+    try:
+        filepath = code.co_filename
+    except AttributeError:
+        filepath = None
+
+    try:
+        ln = code.co_firstlineno
+    except AttributeError:
+        ln = None
+
```
Comment (cknd) [inline stackprinter/utils.py:49]: Hi, thanks for the contrib! Just two tweaks:
- could you shorten this to `getattr(code, 'co_filename', None)` and `getattr(code, 'co_firstlineno', None)`
- and add a comment that explains why this is done - i.e. in which situation these attributes can be None (under pypy, I understand?)

**Review by cknd — CHANGES_REQUESTED** [review]

File: stackprinter/utils.py (hunk 1 above)
Comment (breezechen) [reply stackprinter/utils.py:49]: image

**Review by cknd — APPROVED** [review]

**Comment by cknd** [comment]
now released in https://github.com/cknd/stackprinter/releases/tag/0.2.9 & published to PyPI, try `pip install stackprinter==0.2.9` 👍
